index.html

```diff
@@ -13,7 +13,7 @@
     <header id="home-header">
       <nav id="toolbar-container">
         <h1 class="logo-placeholder">My Logo</h1>
-        <button class="burger"><i class="fa fa-bars"></i></button>
```

Comment:
I tried using the link to the icon library in the <head> and appending the icons to the buttons, but it didn't work well.

This is the lesson I tried to use:
https://www.w3schools.com/howto/howto_css_icon_buttons.asp

Then I decided to use the icon from Figma templates. I put it as a background-image in CSS.